### PR TITLE
Added work section

### PR DESCRIPTION
**Pull request by mansa-susa**
*Added Work Section*
- used grid for the work section container
- used flexbox to layout individual cards

*Added About-me Section*
- used flexbox to layout the language, framework and skills list

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -34,6 +38,141 @@ <h3>I'm a Software Engineer</h3>
         <li><a href="#"><i class="fa fa-medium" aria-hidden="true"></i></a></li>
       </ul>
     </section>
+    <section class="work-section">
+      <div class="recent-works">
+        <h1>My Recent Works</h1>
```
Comment (rotimiazeez) [inline index.html]: - [x] Kindly make sure you don't have more than one `h1` tag on a page as it's a bad practice and it confuses the search engine when searching up your page.😉

File: index.html [hunk 2]
```diff
@@ -34,6 +38,141 @@ <h3>I'm a Software Engineer</h3>
...
+        <ul>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="submit">See Project</button>
+      </article>
+    </section>
+     <section class="about-me">
+      <h1>About me</h1>
```
Comment (rotimiazeez) [inline index.html]: As stated early kindly make sure you don't have more than one h1 tag on a page 👍.

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @mansa-susa  👋, 
## Good job so far!
### Summary 👏
- No linter errors ✔
- Good commit message 👍
- Descriptive PR message😊
- Good Readme 🚀
### There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mansa-susa) [reply index.html]: fixed the changes

**Review by Mre55 — APPROVED** [review]
**Hi @mansa-susa,**

# Status: APPROVED 🏆 🎉

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**